src/client/webpackEntry.js

```diff
@@ -0,0 +1,10 @@
+/*
+ * Setup webpack runtime public path loading, used for configuring
+ * asset loading from our CDN:
+ */
+
+/* eslint-disable camelcase, no-undef */
+const getWebpackPublicPath = require('universal/utils/getWebpackPublicPath');
+__webpack_public_path__ = getWebpackPublicPath();
```

Comment:
N.B. we must put them in their own entry file. When I added this statement to `client.js` the `import` statements were getting hosted above the `__webpack_public_path__ = ` expression, and then the magic ✨  didn't work 🚫 .